src/render.rs

```diff
@@ -211,7 +222,7 @@ where
     factory
         .link_program(code_vs.as_bytes(), code_ps.as_bytes())
         .map_err(|e| {
-            error!(
+            println!(
```

Comment:
why this?

Reply:
Because I faced runtime errors a few times - and `error` doesn't print anything while `println` does. But yep, it was temporal change just for me, I'll revert